.github/workflows/build-app.yml

```
@@ -0,0 +1,99 @@
+name: Build App
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)